libs/core/src/global/styles/tokens/core/_core.scss

```diff
@@ -12,7 +12,7 @@
   --pine-border-radius-050: 4px;
   --pine-border-radius-075: 6px;
   --pine-border-radius-round: 9999px;
-  --pine-border-width-none: 0;
+  --pine-border-width-none: 0rem;
```

Comment:
Curious to understand why 0 needs a unit?

Reply:
@pixelflips This is part of the built-in CSS transform group in Style Dictionary that will add units to 0 values. I just pushed a change that will make sure these values are pixels instead.

Reply:
@anechol gotcha. I was just curious since 0 is typically unitless, but thanks for clarifying it's needed for Style Dictionary. Seems like an odd thing on their part but 🤷🏼.